ambari-web/app/templates/wizard/step3.hbs

```diff
@@ -19,141 +19,159 @@
 <div id="confirm-hosts" class="wizard-content col-md-9">
   <h4 class="step-title">{{t installer.step3.header}}</h4>
   {{#if isSaved}}
+    <div class="alert alert-warning" role="alert"><strong>{{t common.warning}}:</strong> {{t installer.warning.changes}}</div>
```

Comment:
Duplicated line?

Reply:
Thanks for pointing this out. Got duplicated while merging.